data/panel.js

```diff
@@ -8,7 +8,13 @@ countEventLoopLags.addEventListener("click", function() {
   self.port.emit("mode-changed", "eventLoopLags");
 });
 var hangThreshold = document.getElementById("hangThreshold");
-hangThreshold.addEventListener("input", function() {
```

Comment:
Need to test it on panel hide.

As for the form, didn't @Uberi say it reloaded the panel when you hit enter? Isn't that not a good thing? (yeah, we could put an action/onsubmit on the form)